tests/e2e/local-run.js

```diff
@@ -0,0 +1,59 @@
+const util = require("util")
+const exec = util.promisify(require("child_process").exec)
+const spawn = require("child_process").spawn
+
+/* start the testnet and the development server and executes the local end to end tests */
+/* you can provide the name of the file to test only one file > for send.spec.js `yarn test:e2e send` */
+const main = async () => {
+  const filter = process.argv[2]
+  console.log("cloning lunie-backend repo")
+  try {
+    await exec("git clone https://github.com/luniehq/lunie-backend.git lunie-backend")
+  } catch (error) { }
```

Comment:
```suggestion
  } catch (error) { console.log("Cloning lunie-backend repo failed. Assuming it is already cloned.") }
```

Reply:
could it fail for other reasons?

Reply:
ah yeah, if credentials are not set. but I wanted to avoid printing a fat error if the error is expected. we could check before if the folder already exists.